README

```diff
@@ -16,10 +16,6 @@ License
 This plugin is licensed under the GNU General Public License v2. See the file
 LICENSE for the complete terms of this license.
 
-The jbimages plugin for Tiny is included in this plugin under its own
-license (currently Creative Commons Attribution 3.0 Unported License); see
-justboil.me for details.
```

Comment:
(I AM VERY HAPPY THIS IS GOING -- IT CONTAINED A SUBSTANTIAL AMOUNT OF ILL-MAINTAINED CODEIGNITER MULCH AND WAS GOING TO GET US IN THE BACK AT SOME POINT)